clarity/src/vm/database/clarity_db.rs

```diff
@@ -611,6 +611,22 @@ impl<'a> ClarityDatabase<'a> {
         self.store.insert_metadata(contract_identifier, key, data);
     }
 
+    /// Set a metadata entry if it hasn't already been set, yielding
+    ///  a runtime error if it was. This should only be called by post-nakamoto
+    ///  contexts.
```

Comment:
Can you help me understand how this error could be triggered?

Reply:
Actually, this function should just be removed -- at first I was going to try to store the computed reward set as metadata, in which case, this function might have been needed. But metadata storage in Clarity is very closely tied to contract instantiation, so it would be really cumbersome to have done things this way.